api/jobs.go

```diff
@@ -15,6 +16,10 @@ import (
 	uuid "github.com/satori/go.uuid"
 )
 
+const (
+	jobStateFailed = "failed"
```

Comment:
would this be better placed in the model - e.g. in a `Failed` method?

Reply:
or in a package like `apierrors` ?

Reply:
maybe for next time

Reply:
I've moved the constant to the model and renamed it to JobStateFailed so that it gets exported. I've also added a JobStateCreated as that is the other possible job state that the service currently uses.

Reply:
seems to also be a reference to literal "failed" on line 285

Reply:
oh and 287 maybe?

Reply:
looks good